src/renderer/components/dock/dock.tsx

```diff
@@ -75,6 +86,22 @@ class NonInjectedDock extends React.Component<Props & Dependencies> {
     this.element?.current.focus();
   };
 
+  nextTab = (direction: Direction = 1) => {
+    const { tabs, selectedTab } = this.props.dockStore;
+    const currentIndex = tabs.indexOf(selectedTab);
+    const nextIndex = currentIndex + direction;
+
+    // check if moving to the next tab is possible.
+    if (direction === 1 && currentIndex!== -1 && nextIndex >= tabs.length) return;
+
+    // check if moving to the previous tab is possible
+    if (direction === -1 && currentIndex!== -1 && nextIndex < 0) return;
```

Comment:
Wouldn't either of these checks be sufficiant, that way they aren't dependent on the argument.

```suggestion
if (currentIndex >= 0 && !(0 <= nextIndex && nextIndex < tabs.length)) {
  return;
}
```

Reply:
Made the check simpler.